### ISSUE TITLE
Does not work with RazorEngine 3.2.0 

### ISSUE THREAD

**Issue by ericjohannsen** [meleze/Meleze.Web#13]
I just tried out Meleze with an MVC 4 solution that uses RazorEngine 3.2.0 (current version as of today).

The generated HTML pages have syntax errors due to handling of JavaScript.  Here's an example:

ORIGINAL:

<script type="text/javascript">
//<![CDATA[
    var cot_loc0 = (window.location.protocol == "https:") ? "https://secure.comodo.com/trustlogo/javascript/cot.js" :
"http://www.trustlogo.com/trustlogo/javascript/cot.js";
    document.writeln('<scr' + 'ipt language="JavaScript" src="' + cot_loc0 + '" type="text\/javascript">' + '<\/scr' + 'ipt>');
//]]>
</script>

OUTPUT:

<script type="text/javascript">var cot_loc0 = (window.location.protocol == "https:") ? "https: "http: document.writeln('<scr' + 'ipt language="JavaScript" src="' + cot_loc0 + '" type="text\/javascript">' + '<\/scr' + 'ipt>');</script>

Additionally, the very first line of my view:

@model MyNamespace.LoginModel

shows a red squiggly under model with the tool tip text "The name 'model' does not exist in the current context"

The idea behind Meleze is fantastic, but unfortunately it fails on the first page (login) of my application.

Question: Is this project still actively maintained?

**Comment by meleze** [comment]
The Meleze MVC4 package also works with MVC5 and the last version of Razor (v3.0).
If you have sample code with RazorEngine, I take a look.

For the @model, you can take a look in the README. You either have to register Meleze in the GAC or move its configuration to web.release.config (otherwise Visual Studio won't find the DLL)

The project is maintained from time to time (especially when I need something for my own projects) and any help is welcome :-)

**Comment by CheloXL** [comment]
@meleze : Did you try the Web.Release.config thing? I'm using Meleze.Web without any issues and to solve the @model with VS I tried that, but it seems that VS doesn't recognizes the transformations to the web.config inside the Views folder. I have to do that manually (swith the factoryType). I would LOVE to have that done automatically...

**Comment by meleze** [comment]
If you use the Web.Release.config, Meleze won't be used by our application in Visual Studio.
You have to publish the application in Release mode to get the Meleze settings taken into account.

To get Meleze and Intellisense working in Visual Studio, you have to register the Meleze DLL to the GAC with "gacutil.exe -i Meleze.Web"

**Comment by CheloXL** [comment]
Yes... that's what I'm talking about... I don't get any transformation, neither on build nor on publish... (again, only inside the Views folder... the web.config on the root always gets transformed correctly). Thank you anyways!!
